packages/web3-eth-accounts/test/fixtures/account.ts

```diff
@@ -24,8 +24,82 @@ export const validPrivateKeytoAccountData: [string, any][] = [
 ];
 
 export const invalidPrivateKeytoAccountData: [any, string][] = [
-	['', 'Invalid value given "". Error: Private key must be 32 bytes.'],
-	[Buffer.from([]), 'Invalid value given "". Error: Private key must be 32 bytes.'],
-	[undefined, 'Invalid value given "undefined". Error: not a valid string or buffer.'],
-	[null, 'Invalid value given "null". Error: not a valid string or buffer.'],
+	['', 'Error: Private key must be 32 bytes.'],
+	[Buffer.from([]), 'Error: Private key must be 32 bytes.'],
+	[undefined, 'Error: not a valid string or buffer.'],
+	[null, 'Error: not a valid string or buffer.'],
+];
+
+export const signatureRecoverData: [string, any][] = [
+	[
+		'Some data',
+		{
+			address: '0xEB014f8c8B418Db6b45774c326A0E64C78914dC0',
+			privateKey: '0xbe6383dad004f233317e46ddb46ad31b16064d14447a95cc1d8c8d4bc61c3728',
+			data: 'Some data',
+			// signature done with personal_sign
+			signature:
+				'0xa8037a6116c176a25e6fc224947fde9e79a2deaa0dd8b67b366fbdfdbffc01f953e41351267b20d4a89ebfe9c8f03c04de9b345add4a52f15bd026b63c8fb1501b',
+		},
+	],
+	[
+		'Some data!%$$%&@*',
+		{
+			address: '0xEB014f8c8B418Db6b45774c326A0E64C78914dC0',
+			privateKey: '0xbe6383dad004f233317e46ddb46ad31b16064d14447a95cc1d8c8d4bc61c3728',
+			data: 'Some data!%$$%&@*',
+			// signature done with personal_sign
+			signature:
+				'0x05252412b097c5d080c994d1ea12abcee6f1cae23feb225517a0b691a66e12866b3f54292f9cfef98f390670b4d010fc4af7fcd46e41d72870602c117b14921c1c',
+		},
+	],
+];
+
+export const transactionsTestData: [any][] = [
```

Comment:
Should not use `any` but the transaction types from `@ethereumjs/tx`

Reply:
not required to validating types in this test

Reply:
It's not required but it helps the developer understand what the expected data is for the tests. Other packages use this ideology and we should remain consistent